package.json

```diff
@@ -26,6 +26,8 @@
   "dependencies": {
     "dotenv": "^6.2.0",
     "openzeppelin-eth": "^2.1.3",
+    "tabookey-gasless": "^0.3.3-a",
+    "truffle": "5.0.4",
```

Comment:
Do we need Truffle here as dependency? Why?

Reply:
removed